### DISCUSSION TITLE
How do I show clip skips in the UI

### DISCUSSION THREAD

**Discussion by flare777** [lllyasviel/stable-diffusion-webui-forge#2013] [General]
I can't get the clip skip UI to display.
Is my setup wrong?
If not, how can I get the clip skip UI to display?
![clip](https://github.com/user-attachments/assets/88404d82-4390-44f3-881b-785a4b2f7a1b)

**Comment by jswag245** [comment]
I believe you have to switch from SD to the other UI settings, like XL, Flux, and All. Should be a the top left.

**Reply by flare777** [reply]
Hello.
The clip skip was only displayed when ALL was selected.
If I use this mode, will there be no problem with SDXL or PONY?
Thank you.

**Reply by jswag245** [reply]
I have been told that the clipskip when using pony/SDXL is automatically changed to 2.

**Reply by flare777** [reply]
> pony/SDXL 使用時のクリップスキップは自動的に 2 に変更されると聞きました。

Thanks for the info.
